src/views/infuse.template.html

```diff
@@ -35,13 +35,15 @@ <h1 translate="Infusion"></h1>
     <div ng-show="vm.infused != 0">
       <dim-simple-item class="infusedItem" item-data="vm.target"></dim-simple-item>
       <div class="result">
-        <span translate="Infusion.BringGear"></span>
+        <div translate="Infusion.BringGear"></div>
         <span class="bigValue">{{vm.infused}}</span>
         <span class="green">(+ {{vm.infused - vm.source.primStat.value}})</span>
-        <span translate="Infusion.Using3"> </span><span class="currency"><img alt="Legendary Marks" title="Legendary Marks" src="~app/images/legendaryMarks.png"></span>
-        <span ng-if="vm.exotic">+ 1 <span class="currency"><img alt="Exotic Shards" title="Exotic Shards" src="https://www.bungie.net/common/destiny_content/icons/e6d2a0561e784ae224e2aa53aa66e8a7.jpg"></span></span>
-        <span>+ 250 <span class="currency"><img alt="Glimmer" title="Glimmer" src="~app/images/glimmer.png"></span></span>
-        <span>+ {{vm.wildcardMaterialCost | number}} <span class="currency"><img alt="Materials" title="Materials" ng-src="{{vm.wildcardMaterialIcon | bungieIcon}}"></span></span>
+        <div>
+          <span> 3 <span class="currency"><img alt="Legendary Marks" title="Legendary Marks" src="~app/images/legendaryMarks.png"></span></span>
```

Comment:
If you can get ahold of the actual item definitions you can use the strings directly.